### PR TITLE
add KSRescue

### PR DESCRIPTION
**Pull request by Zohar97**

### CODE REVIEW

File: contracts/KSRescue.sol [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  using SafeERC20 for IERC20;
+
+  address private constant ETH_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
+
+  function rescueFunds(IERC20 token, uint256 amount, address recipient) external onlyOwner {
+    if (amount == 0) amount = _getAvailableAmount(token);
+    if (amount > 0) {
+      if (_isETH(token)) {
+        (bool success,) = recipient.call{value: amount}("");
+        require(success, 'FundSecure: ETH_TRANSFER_FAILED');
```
Comment (manhlx3006) [inline contracts/KSRescue.sol]: `KSRescue: ETH_TRANSFER_FAILED`?

Comment (Zohar97) [reply contracts/KSRescue.sol]: fixed

File: contracts/KSRescue.sol [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
+
+abstract contract KSRescue is KyberSwapRole {
+  using SafeERC20 for IERC20;
+
+  address private constant ETH_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
+
+  function rescueFunds(IERC20 token, uint256 amount, address recipient) external onlyOwner {
+    if (amount == 0) amount = _getAvailableAmount(token);
```
Comment (danielpham765) [inline contracts/KSRescue.sol:15]: Should add one more check:
```require(recipient != address(0), 'KSRescue: recipient == address(0)');```

Comment (Zohar97) [reply contracts/KSRescue.sol:15]: fixed

**Pull request merged**